tests/test_grid.py

```diff
@@ -1072,6 +1072,16 @@ def test_hide_points(ind, struct_grid):
         struct_grid.hide_points(np.ones(10, dtype=np.bool))
 
 
+def test_set_extent():
+    uni_grid = pyvista.UniformGrid(dims=[10, 10, 10])
+    with pytest.raises(ValueError):
+        uni_grid.extent = [0, 1]
+
+    extent = [0, 1, 0, 1, 0, 1]
+    uni_grid.extent = extent
+    assert np.allclose(uni_grid.extent, extent)
```

Comment:
My usual recommendation if these are ints
```suggestion
    assert np.array_equal(uni_grid.extent, extent)
```